dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -381,6 +381,7 @@ If you have providers as list of provider ids because you just released them, yo
 ```
 
 - Copy the documentation to the ``airflow-site`` repository
+- Either provide the ``--airflow-site-directory`` flag or set the env variable: ``AIRFLOW_SITE_DIRECTORY``
```

Comment:
AIRFLOW_SITE_DIRECTORY is always point to site directory it's part of release process. I'm not sure what is the value of this line?

Reply:
Yeah, doesn't make sense anymore. There were some earlier discussions leading to this decision